third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -276,9 +276,15 @@ function commandCallback(method, params) {
   }
 
   try {
+    // track performance temporarily (RUN-1315)
+    const startTime = Date.now();
     VerboseCommands && log(`[Command ${method}] Handling command, params=${JSON_stringify(params)}...`);
     const result = CommandCallbacks[method](params);
     VerboseCommands && log(`[Command ${method}] Handled command, result=${JSON_stringify(result)}`);
+    const timeTaken = (Date.now() - startTime)/1000;
+    if (timeTaken > 0.2) {
+      log(`[RuntimeWarning][Command ${method}] slow: ${timeTaken.toFixed(2)}s, params: ${JSON.stringify(params)}`);
+    }
```

Comment:
This should be removed, the recorder also tracks the time taken and we can get the information from those logs.  In particular we can't log the params as they can contain sensitive user information.

Reply:
Done x3